### PR TITLE
feat : added Instruction page

### PR DESCRIPTION
**Pull request by Priya-Patel31**
* Just changed the html into jsx of rules page .

### CODE REVIEW

File: src/services/homePage_services.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import data from "../utils/trophy.json"
+
+export const getCategories = ()=>{
+    console.log(data)
```
Comment (Ankur9669) [inline src/services/homePage_services.js:4]: Can remove console.log statement here

File: src/utils/fetchApi.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import {axios} from "react"
+export async function fetchApi(url,methodName,body,propertyName){
+try{
+    const {data,statusCode} = await axios({methodName})
+    if(statusCode === 200 || statusCode === 201){
+        console.log(data);
```
Comment (Ankur9669) [inline src/utils/fetchApi.js:6]: Same

**Review by Ankur9669 — APPROVED** [review]
The coding conventions are nice looks good to merge

**Review by krate27 — APPROVED** [review]
Looks good to go!
Few suggestions though:
1. Use consistent units.
2. CSS could be more optimized, just a suggestion.

Rest great work.

File: src/services/homePage_services.js (hunk 1 above)
Comment (Priya-Patel31) [reply src/services/homePage_services.js:4]: done

File: src/utils/fetchApi.js (hunk 2 above)
Comment (Priya-Patel31) [reply src/utils/fetchApi.js:6]: done

**Pull request merged**